### ISSUE TITLE
Gauth profile disappeared

### ISSUE THREAD

**Issue by comhmis** [gbraad/gauth#94]
When I add an account, the account disappears upon the next time  I open the Gauth app on my computer in Chrome. It is for Looker and I'm noticing other users are having the same problem. Is there a way to retain the account in Gauth desktop version?  It seems like the mobile app is working fine, but the desktop app isn't retaining added accounts. Thanks.

**Comment by gbraad** [comment]
What do you mean with desktop version?

There are basically two supported version;
  * Website: https://gauth.apps.gbraad.nl
  * FirefoxOS: from the mozilla store.

Note: The current version of the Chrome extension merely wraps the https:// endpoint.

If the account is disappearing, this is likely related to a plugin or cookie cleaning process. Although the tokens are stored in LocalStorage, they are cleaned/purged by Chrome when a cookie cleaning action is triggered.

Would it be possible to provide more details: what version of Chrome are you running? Which platform are you using; Windows, Linux or Mac OS? Where did you get the 'desktop version' you refer to?

**Comment by comhmis** [comment]
Yes, the desktop version.
I’m using Windows 7 Enterprise (with Service Pack 1) in a networked environment. Using the Version 53.0.2785.116 m (64-bit) Chrome. I went to the Chrome app store, installed the app and created shortcuts on my desktop, toolbar, and I have it from my Apps page [cid:image003.png@01D2131E.B06D37A0]  in Chrome. I tried using each of these methods. When I close GAuth the added account is gone the next time I open it without closing Chrome and without clearing out cookies. My process is to open the app, add my Looker account, exit/close Chrome, exit from the app, and re-open Chrome and the app--- the Looker account I just added is gone.

Should I try using Firefox?
Thanks so very much for getting back to me too,
Maylee
